README.md

```diff
@@ -1,99 +1,99 @@
 
-## Contents
+## Introduction
 
-* [Prerequisites](#Prerequisites)
-* [Getting Started](#Getting-Started)
-  * [Master Node(s)](#Master-Node(s))
-  * [Worker Node(s)](#Worker-Node(s))
-* [Contributing](#Contributing)
-* [Authors](#Authors)
-* [License](#License)
-* [Acknowledgments](#Acknowledgments)
+This guide will walk through the steps required to bootstrap a running Kubernetes cluster with a highly available topology. You will learn and configure the CRI and CNI of your choice (assuming it is supported) and understand how to setup load balancing between your multi master cluster.
 
-## Prerequisites
-
-Prior to getting started you must have completed the following setup instructions:
-
-The SD card must be flashed with Raspbian Lite. If this has not been completed the official guide can be found [HERE](https://www.raspberrypi.org/documentation/installation/installing-images/README.md).
+> Note: If you wish to use Raspbian Lite please use the following [guide](raspbian/README.md).
 
-Note: To setup SSH to your Raspberry Pi, you simply need a empty `ssh` file in the boot directory.
-
-You must also have the following dependencies installed on your machine:
+## Prerequisites
 
-* make
-* sshpass
+Prior to getting started you will need to have several things done first. Assuming you already have the [hardware](#Hardware) available, you will need to do the following:
 
-An internet connection is also required for configuring the Raspberry Pi's.
+1. Flash OS onto SD card. Guide [Here](setup/README.md)
+2. Install Ansible. Download [Here](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+3. SSH connectivity to each node. (Step 1 requires you setup SSH keys)
 
 ## Getting Started
 
-When configuring a Raspberry Pi there are two options for configuring it as a node in your Kubernetes cluster, it can either be a `master` or `worker`.
+Once the Raspberry Pi's are running and all the prerequisites have been completed we're now ready to setup the Ansible inventory.
 
-At this point the Raspberry Pi should be running and on the same network as your machine so it can be accessed over ssh.
+Open the [inventory file](ansible/inventory) - each machine that will be joining the Kubernetes cluster must be defined as either a master or worker node. To leverage the highly available topology configuration you would ideally have 3 masters available as a minimum, otherwise 1 master node is fine, however, it won't be highly available.
 
-### Master Node(s)
+> Note: Ensure the `hostname` matches what the machine was given when flashed the SD card and `ansible_host` matches the IP allocated to the host on your prefered subnet.
 
-The master node configuration is copied from your machine to the Raspberry Pi with the following command:
+When the inventory has been configured with all the hosts that will be joining the Kubernetes cluster we can run the following command to verify SSH connectivity can be established.
 
```

Comment:
A link to a sample/current inventory would be helpful here

Reply:
`[inventory file](ansible/inventory)`